public/instructors_list.php

```diff
@@ -56,6 +56,11 @@
     header("refresh: 1");
 }
 
+if (isset($_POST['delete_association']) && LectureInstructor::find_by(array('lecture_id' => $_POST['lecture_id'], 'user_id' => $_POST['instructor_id'])) != null) {
+    LectureInstructor::find_by(array('lecture_id' => $_POST['lecture_id'], 'user_id' => $_POST['instructor_id']))->deleteWhere('user_id', 'lecture_id');
+    header("refresh: 1");
```

Comment:
I approve of the idea of implementing a delete_where (but without fetching the records first)

Reply:
not sure how to do this :S 
It works the way it is now

Reply:
where is the function? can't find it

Reply:
it should be in records just after the delete function

Reply:
huh it wasnt added again weird

Reply:
added it